AnimateSvg.js

```diff
@@ -0,0 +1,119 @@
+import React, { useRef, useEffect, useState } from 'react';
+import { Animated, Text, View } from 'react-native';
+import { useIsShake } from './useIsShake';
+import { SVG_LOAD_DURATION, SVG_OUT_DURATION, SVG_IN_DURATION, SVG_IN_DELAY, QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, TRANSITION_TIME_2 } from './constants';
+
+export default function AnimateSvg(props) {
+  const [isRegistered, setIsRegistered] = useState(false);
+  const [hasToggledBefore, setHasToggledBefore] = useState(false);
+
+  const fadeAnim = useRef(new Animated.Value(0)).current;
+  const spinValue = useRef(new Animated.Value(0)).current;
+  const spin = spinValue.interpolate({
+    inputRange: [0, 1],
+    outputRange: ['0deg', '1080deg'],
+  });
+  const sizeAnim = useRef(new Animated.Value(1)).current;
+
+  const { toggle } = useIsShake();
+
+  useEffect(() => {
+    Animated.timing(fadeAnim, {
+      toValue: 1,
+      duration: SVG_LOAD_DURATION,
+      useNativeDriver: true,
+    }).start();
+  }, []);
+
+  // LogoSvg fades out first time
+  if (toggle === true && hasToggledBefore === false && isRegistered === false) {
+    console.log('AnimateSvg, 1st if block, beginning');
+    setIsRegistered(true);
+    setHasToggledBefore(true);
+    Animated.parallel([
+      Animated.timing(fadeAnim, {
+        toValue: 0,
+        duration: SVG_OUT_DURATION,
+        useNativeDriver: true,
+      }),
+      Animated.timing(spinValue, {
+        toValue: 1,
+        duration: SVG_OUT_DURATION,
+        useNativeDriver: true,
+      }),
+      Animated.timing(sizeAnim, {
+        toValue: 0.5,
+        duration: SVG_OUT_DURATION,
+        useNativeDriver: true,
+      }),
+    ]).start();
```

Comment:
As per our conversations, `.start()` takes a callback to run after the animation finishes. Address in a followup